hrm-service/src/config/ssmCache.ts

```diff
@@ -0,0 +1,14 @@
+import { loadSsmCache as loadSsmCacheRoot } from '@tech-matters/hrm-ssm-cache';
+
+export { getSsmParameter, hasCacheExpired, ssmCache } from '@tech-matters/hrm-ssm-cache';
+
+const ssmCacheConfigs = [
+  {
+    path: `/${process.env.NODE_ENV}/sqs/jobs/contact`,
```

Comment:
Is `NODE_ENV` the appropriate env var here? The values for this can be test, development and production I believe (unless we are manually setting them somewhere else?).

Just asking to understand.

Reply:
They are being set in the hrm-secrets env files to match the environment so `staging` is also an option. That is what makes [this work in the model config](https://github.com/techmatters/hrm/blob/master/hrm-service/src/models/index.js#L24). #253 includes changes to pull a .env for hrm-service from ssm. In that .env we set `NODE_ENV=local` to match the behavior of the tasks running in ECS.